purchases/src/test/java/com/revenuecat/purchases/common/backend/BackendTest.kt

```diff
@@ -50,6 +51,7 @@ import io.mockk.unmockkObject
 import io.mockk.verify
 import org.assertj.core.api.Assertions.assertThat
 import org.assertj.core.api.Fail.fail
+import org.checkerframework.checker.optional.qual.Present
```

Comment:
Is this intended?

Reply:
Nope, unused import... We should run detekt in the tests as well, but that will probably be a lot of changes 😅